voucher/src/main/java/org/prgms/voucher/application/CommandLineApplication.java

```diff
@@ -0,0 +1,48 @@
+package org.prgms.voucher.application;
+
+import org.prgms.voucher.voucher.VoucherController;
+import org.prgms.voucher.voucher.VoucherPrintDto;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class CommandLineApplication implements Console, OutPut {
+    private static final String VOUCHER_TYPE = "바우처 종류";
+    private static final String AFTER_DISCOUNT_VALUE = "할인가";
+    private static final String PUBLISH_DATE = "발행일";
+    private static final String EXPIRATION_DATE = "만료일";
+    private static final String SEPARATOR = ": ";
+    private final VoucherController voucherController;
+
+    public CommandLineApplication(VoucherController voucherController) {
+        this.voucherController = voucherController;
+    }
+
+    @Override
+    public void printSupportedCommands() {
+
+    }
+
+    @Override
+    public void exitProgram() {
+
+    }
+
+    @Override
+    public String printList() {
+        List<VoucherPrintDto> voucherPrintDtos = voucherController.listVoucher();
+
+        StringBuilder sb = new StringBuilder();
+
+        for (VoucherPrintDto voucherPrintDto : voucherPrintDtos) {
+            sb.append("\n");
+            sb.append(VOUCHER_TYPE).append(SEPARATOR).append(voucherPrintDto.getOptionType()).append("\n");
```

Comment:
```suggestion
            sb.append(VOUCHER_TYPE + SEPARATOR + voucherPrintDto.getOptionType() + "\n");
```
이게 좀 더 가독성이 좋을 것 같아요.

Reply:
해당 부분 `stream`과 `+`를 사용하여 반영 완료했습니다.
[refactor: 리스트 출력하는 printList메서드 수정](https://github.com/prgrms-be-devcourse/springboot-basic/pull/715/commits/e00352444cab6d95bce926901a3276f3c3d4a0ca)